### DISCUSSION TITLE
Remove bumptech.glide.load.data java package from maui app

### DISCUSSION THREAD

**Discussion by som-nitjsr** [dotnet/maui#23686] [Q&A]
### Description

I am building a maui app  targeting .Net 8.
I see the app has dependency on java package bumptech.glide.load.data which allow HTTP(non https) URL also.
i could not find direct depnedcny on bumptech.glide.load.data package what is the best way to remove it.

### Steps to Reproduce

1.  Create a .net Maui app 
2.  build the android app
3. open the classes.dex file from app 

### Version with bug

8.0.3 GA

### Is this a regression from previous behavior?

Not sure, did not test other versions

### Last version that worked well

Unknown/Other

### Affected platforms

Android

### Affected platform versions

all version

### Did you find any workaround?

no

**Comment by PureWeen** [comment]
Everything that MAUI does with images on Android is via glide
